### ISSUE TITLE
Considered using kubernetes_manifest resource?

### ISSUE THREAD

**Issue by sepulworld** [WOGRA-AG/terraform-kustomization-kubeflow#3]
Hi! 👋 Great project and thank you for sharing.

Quick question - have you all considered moving away from kubectl_manifest provider to instead use the Kubernetes provider [kubernetes_manifest](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest
) resource? It will probably have better long term support and might handle resource dependencies better. Thoughts?

**Comment by pd-t** [comment]
Yes, you are completely right and we already realized it. We just had not time to refactor the provider. At the moment, we also give Argo-cd a try and I have to admit that it feels quite good. So, perhaps we end up providing files for argocd support and setup everything with terraform. The point is, that this mixed setting is faster and is much more stable to changes.

**Comment by pd-t** [comment]
@sepulworld One more question to you. From a user perspective, would you like to have a terraform kustomization native module or would you accept an argocd mixing?

**Comment by tom333** [comment]
If it can help, I've made an exemple of app of apps argocd's pattern
it's available here if you want : https://github.com/tom333/argocd-kubeflow

**Comment by pd-t** [comment]
Thanks a lot! We are currently refactoring the deployment of our main project os4ml. So, I hope that we can give an update to the end of september on this issue.
